tests/php/ModelAdminTest.php

```diff
@@ -37,6 +38,17 @@ public function testModelAdminOpens()
         $this->assertEquals(200, $this->get('ContactAdmin')->getStatusCode());
     }
 
+    public function testMultiModelAdminOpensEachTab()
```

Comment:
What's this thing meant to do?

It's [failing our kitchen sink build](https://travis-ci.com/github/silverstripe/cwp-recipe-kitchen-sink/jobs/510738944) and it's not actually making any assertions.

Reply:
Previously, this would throw an exception in the `doInit()` call by hitting: https://github.com/silverstripe/silverstripe-admin/blob/1/code/ModelAdmin.php#L162

This "asserts" that each tab will open without crashing in the MultiModelAdmin.

Not sure why that would fail in the kitchen sink but pass here - my guess is it would be fixed by setting the session in the same way as in `testExportFieldsDefaultIsSummaryFields()`?

Reply:
Tried defining a session like the other test, but got a different error.

```
Already directed to http://localhost/Security/login?BackURL=%2F; now trying to direct to Security/login?BackURL=%2F
 /home/maxime/Projects/cwp2x/vendor/silverstripe/framework/src/Control/Controller.php:640
 /home/maxime/Projects/cwp2x/vendor/silverstripe/admin/code/LeftAndMain.php:827
 /home/maxime/Projects/cwp2x/vendor/silverstripe/framework/src/Security/Security.php:436
 /home/maxime/Projects/cwp2x/vendor/silverstripe/subsites/src/Extensions/LeftAndMainSubsites.php:372
 /home/maxime/Projects/cwp2x/vendor/silverstripe/framework/src/Core/Extensible.php:468
 /home/maxime/Projects/cwp2x/vendor/silverstripe/framework/src/Control/Controller.php:123
 /home/maxime/Projects/cwp2x/vendor/silverstripe/admin/tests/php/ModelAdminTest.php:49
 /home/maxime/Projects/cwp2x/vendor/sminnee/phpunit/phpunit:52
```

All good ... I'll try to have a go at fixing it.

Reply:
Thanks mate, let me know how it goes